src/app/hubspot/api.ts

```diff
@@ -0,0 +1,86 @@
+import { Client } from "@hubspot/api-client"
+import "server-only"
+import { HubspotLegacyProfile } from "./hubspotModels"
+
+export const hubspotClient = new Client({
+  accessToken: process.env.HUBSPOT_API_KEY,
+})
+
+const hubspotSummitProperties = [
+  "first_time_attendee_",
+  "i_m_attending",
+  "registering_as",
+  "what_information_or_questions_would_you_like_covered_at_this_summit_",
```

Comment:
Maybe extract these strings to some object/enum as you're using them in multiple places.